cmd/deployment/create.go

```diff
@@ -185,7 +185,7 @@ func initFlags() {
 	createCmd.Flags().String("es-ref-id", "main-elasticsearch", "Optional RefId for the Elasticsearch deployment")
 	createCmd.Flags().Int32("es-zones", 1, "Number of zones the Elasticsearch instances will span")
 	createCmd.Flags().Int32("es-size", 4096, "Memory (RAM) in MB that each of the Elasticsearch instances will have")
-	createCmd.Flags().StringArrayP("topology-element", "e", nil, "Optional Elasticsearch topology element definition. See help for more information")
+	createCmd.Flags().StringArrayP("es-node-topology", "e", nil, "Optional Elasticsearch node topology element definition. See help for more information")
```

Comment:
most of the flags are set as `StringSlice` not `StringArray`. I think we should change this to be `StringSliceP`. I know it's not related to your change, but I'm just seeing it now.

Reply:
The command doesn't work with `StringSlice` unfortunately :( I tried the command changing both lines you mentioned and I get `invalid argument "{\"size\": 1024, \"zone_count\": 2, \"node_type\": \"data\"}" for "-e, --es-node-topology" flag: parse error on line 1, column 1: bare " in non-quoted-field`

I think it's best to leave them as is, otherwise I feel like I might have to change several things. What do you think?

Reply:
Ah that's right, that's why it's an array. sorry for the noise!